src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/Builders/PublicationServiceMockBuilder.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using GovUk.Education.ExploreEducationStatistics.Common.ViewModels;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using GovUk.Education.ExploreEducationStatistics.Content.Requests;
+using GovUk.Education.ExploreEducationStatistics.Content.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Content.ViewModels;
+using Moq;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Services.Tests.Builders;
+
+public class PublicationServiceMockBuilder
```

Comment:
I guess this whole `PublicationServiceMockBuilder` is not used now, so could be removed, but worth leaving in?